lang/ru.xml

```diff
@@ -186,31 +206,36 @@ depending on its size and your hardware.</app_delete_note>
 		<minutes_ago>{} мин. назад</minutes_ago>
 	</common>
 
-	<compatibility name="Совместимости">
+	<compatibility name="Совместимость">
 		<states>
-			<state id = "-1">Unknown</state>
-			<state id = "0">Бесполезно</state>
-			<state id = "1">Загружается</state>
-			<state id = "2">Интро</state>
+			<state id = "-1">Неизвестно</state>
+			<state id = "0">Не запускается</state>
+			<state id = "1">Запускается</state>
+			<state id = "2">Заставка</state>		
 			<state id = "3">Меню</state>
 			<state id = "4">В игре -</state>
 			<state id = "5">В игре +</state>
 			<state id = "6">Играбельно</state>
 		</states>
 	</compatibility>
 
+	<compile_shaders>
+		<compiling_shaders>Компиляция шейдеров</compiling_shaders>
+		<shaders_compiled>{} шейдеров скомпилировано</shaders_compiled>
+	</compile_shaders>
+
 	<content_manager name="Управление данными">
-		<search>Search</search>
-		<application name="Приложения">
-			<delete>Выбранные приложения и все связанные с ними данные, в том числе сохраненные данные, будут удалены.</delete>
+	<search>Поиск</search>
+		<application name="Приложение">
+			<delete>Выбранные приложения и все связанные с ними данные, в том числе сохранённые данные, будут удалены.</delete>
 			<no_item>Отсутствуют элементы данных.</no_item>
 		</application>
-		<saved_data name="Сохраненные данные">
-			<delete>Выбранные сохраненные данные будут удалены.</delete>
-			<no_saved_data>Отсутствуют сохраненные данные.</no_saved_data>
+		<saved_data name="Сохранённые данные">
+			<delete>Выбранные сохранённые данные будут удалены.</delete>
+			<no_saved_data>Отсутствуют сохранённые данные.</no_saved_data>
 		</saved_data>
 		<theme>Тема</theme>
-		<free_space>Cвободное</free_space>
+		<free_space>Cвободно</free_space>
```

Comment:
revert all

Reply:
no need revert, is all right

Reply:
> no need revert, is all right

i uysing official psvita string, so is wrong

Reply:
please could you show me where the string is used officially? on discord if preferred

Reply:
? i just said here, is officiel string from psvita